src/aat/resources/features/F-063/S-214.td.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "must return 404 when request provides authentic credentials without authorized access to the operation",
+
+  "_guid_": "S-214",
+  "_extends_": "F-063_Base_Test_Data",
+
+  "specs": [
+    "an active profile in CCD",
+    "contains an authentication token for a user that does not have read access to the case type",
+    "contains a HTTP 404 Not Found"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "ctid": "AAT_AUTH_1"
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_404_Response",
+    "headers": {
+      "Content-Encoding": "gzip",
+      "Content-Length": "[[ANY]]",
+      "Content-Type": "application/json;charset=UTF-8"
+    },
```

Comment:
can we move "Content-Length": "[[ANY]]",
      "Content-Type": "application/json;charset=UTF-8" to base file

Reply:
moved to base test data file 👍